.github/workflows/publish.yaml

```diff
@@ -0,0 +1,28 @@
+name: Publish Package to Github Packages 
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment: publish
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://npm.pkg.github.com'
+          scope: '@swan-bitcoin'
+      - run: yarn install --frozen-lockfile
+      - run: yarn workspaces run compile
+      # publish xpub-lib
+      - run: yarn publish xpub-lib --access public
```

Comment:
I see that `yarn publish` expects a specific folder as arg. Is the current command going to work correctly?

https://classic.yarnpkg.com/lang/en/docs/cli/publish/#toc-yarn-publish-folder

Alternative:
```
yarn publish packages/xpub-lib --access public
```

Reply:
Good catch, will update the folder path for both.